lefthook.yml

```diff
@@ -3,5 +3,5 @@ pre-commit:
   commands:
     prettier:
       description: "Check for prettier formatting"
-      run: pnpm run prettier:check {staged_files}
```

Comment:
`{staged_files}` is cool, but I'm not able to exclude .sh scripts from that pattern, as prettier doesn't work for shell files